### PR TITLE
edited provider-quantinuum with link to latest technical specs

### PR DESCRIPTION
**Pull request by mlk621**
edited provider-quantinuum to provide link to most up to date technical specs, fixed references for H-System Quantum Credits, updated a honeywell target reference in credits-faq

### CODE REVIEW

**Comment by PRMerger-2** [comment]
@mlk621 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger3** [comment]
@mlk621 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: articles/provider-quantinuum.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Billing information: H1 Emulator usage is offered free-of-charge with a hardware
 
 ## System Model H1
 
-Quantinuum's quantum computer, the System Model H1, Powered by Honeywell, includes two hardware machine targets: H1-1 and H1-2.  Both machines have fundamentally the same design and both meet a nominal set of technical requirements. However, they may have system-to-system variability in exact performance and features, such as the maximum number of available qubits. Users are encouraged to test compatibility of their code by submitting jobs to the [Syntax Checker](#syntax-checker) prior to submitting them to the target machines.  
+The System Model H1 generation of quantum computers, Powered by Honeywell, includes two machine targets: H1-1 and H1-2. Both quantum computers have fundamentally the same design and both meet a nominal set of technical requirements. However, they may have system-to-system variability in exact performance and features, such as the maximum number of available qubits. Users are encouraged to test compatibility of their code by submitting jobs to the [Syntax Checker](#syntax-checker) and [System Model H1 Emulator](#system-model-h1-emulator) prior to submitting them to the target machines.
```
Comment (Bradben) [inline articles/provider-quantinuum.md:65]:
```suggestion
The System Model H1 generation of quantum computers, Powered by Honeywell, includes two machine targets: H1-1 and H1-2. Both quantum computers have fundamentally the same design and both meet a nominal set of technical requirements. However, they may have system-to-system variability in exact performance and features, such as the maximum number of available qubits. Users are encouraged to test compatibility of their code by submitting jobs to the [syntax checker](#syntax-checkers) and [System Model H1 Emulator](#system-model-h1-emulators) prior to submitting them to the target machines.  
```

File: articles/provider-quantinuum.md [hunk 2]
```diff
@@ -76,21 +76,9 @@ Both System Model H1 hardware H1-1 and H1-2 are continuously upgraded throughout
...
-- Typical limiting fidelity >99.5% (two-qubit fidelity)
-- Coherence Time ($T_2$) ~3 sec
-- Ability to perform mid-circuit measurement and qubit reuse
-- High-resolution rotations (> $\pi$/500)
-- Native Gate set:
-  - single-qubit rotations
-  - two-qubit ZZ-gates
-
-More details available under NDA.
+Technical details for System Model H1 and System Model H1 Emulators are found in Quantinuum's Product Data Sheets, which can be found here: [Quantinuum System Model H1](https://www.quantinuum.com/products/h1).
```
Comment (Bradben) [inline articles/provider-quantinuum.md:81]:
```suggestion
Technical details for the System Model H1 and System Model H1 Emulators can be found in Quantinuum's [product data sheets](https://www.quantinuum.com/products/h1).
```

**Review by Bradben — APPROVED** [review]
Just some minor copyedits - otherwise, it looks good. Thanks!

**Comment by jborsecnik** [comment]
Hi @guenp, @Bradben, and @SoniaLopezBravo

Can you review the proposed changes? Let us know if you can approve all the changes or only those for specific articles. 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. Thanks.

#label:"aq-pr-triaged" 
#assign:@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger8** [comment]
The following users don't have valid GitHub IDs, or are not collaborators on this repo: microsoftdocs/public-repo-pr-review-team

**Comment by PRMerger7** [comment]
@Bradben : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Bradben — APPROVED** [review]

**Comment by Bradben** [comment]
#sign-off

**Pull request merged**